ros2pkg/ros2pkg/verb/create.py

```diff
@@ -0,0 +1,112 @@
+# Copyright 2017 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import getpass
+import os
+
+from ros2pkg.verb import VerbExtension
+from ros2pkg.api.create_api import create_folder
+from ros2pkg.api.create_api import create_template_file
+
+class CreateVerb(VerbExtension):
+    """Output the prefix path of a package."""
+
+    def add_arguments(self, parser, cli_name):
+        arg = parser.add_argument(
+            'package_name',
+            help='The package name')
+        arg = parser.add_argument(
+            '--destination-directory',
+            default=os.path.realpath(os.getcwd()),
+            help='Directory where to create the package')
+        arg = parser.add_argument(
+            '--build-tool',
+            default='cmake',
+            choices=['cmake', 'ament-cmake'],
+            help='Which build tool to use')
```

Comment:
"build tool" is the wrong term here. "The build type of the package" would be more accurate.